test-app/tests/integration/components/changeset-test.gts

```diff
@@ -0,0 +1,222 @@
+/* eslint-disable no-undef -- Until https://github.com/ember-cli/eslint-plugin-ember/issues/1747 is resolved... */
```

Comment:
in a follow up PR, I think it'd be good to test scenarios which use these as well: https://github.com/poteto/ember-changeset-validations

Reply:
Like, I was expecting to see these:
```
import {
  validatePresence,
  validateLength,
  validateConfirmation,
  validateFormat
} from 'ember-changeset-validations/validators';
```

Reply:
Yeah, I can do that. However those already use the underlying APIs of ember-changeset that we are using here. So I don't expect the tests to really increase the test coverage, other than testing ember-changeset-validations itself. But maybe it's good to have that nevertheless, for the teaching-aspects of tests (yeah, tests have that! 😀), as they show a more realistic use case?